### ISSUE TITLE
integration/k8s: unify on a single version of nginx

### ISSUE THREAD

**Issue by grahamwhaley** [kata-containers/tests#1693]
**Which feature do you think can be improved?**

Post running the integration/k8s tests, examining the image pools shows that we currently pull both `nginx:latest` and `nginx:1.14`. It looks like we don't have to pin (and generally we don't like pinning).

**How can it be improved?**

Don't pin it. Using the same version will save us a bunch of download time and space (~113Mb)

**Comment by chavafg** [comment]
I think we had `1.14`[1] pinned because we once had an issue with the `latest` tag on docker hub, so I would recommend to just update the version to something the newest working version which seems to be `1.17` (didn't test it)... wdyt? 
/cc @GabyCT 

[1] https://github.com/kata-containers/tests/blob/master/versions.yaml#L33-L36

**Comment by grahamwhaley** [comment]
I did wonder if we pinned for a reason (but did not have time to try and trawl the github history).
If we need to pin, do we think we can do it via the local versions.yaml file, and use that for all nginx based tests? At least that way it is clear we pinned, only stored in one place, and easy to update later. I guess that would need some in-flight dynamic .yaml file editing though (but I think we do that in some other tests already??)

**Comment by chavafg** [comment]
right, we already do some editing using `sed` and as you mention, we have forgotten to reuse it in some places... I'll prefer to have it pinned so we don't get surprises later on
